ee/hogai/taxonomy_agent/toolkit.py

```diff
@@ -325,7 +331,7 @@ def retrieve_event_property_values(self, event_name: str, property_name: str) ->
         except PropertyDefinition.DoesNotExist:
             return f"The property {property_name} does not exist in the taxonomy."
 
-        runner = EventTaxonomyQueryRunner(EventTaxonomyQuery(event=event_name), self._team)
+        runner = EventTaxonomyQueryRunner(EventTaxonomyQuery(event=self._normalize_event_name(event_name)), self._team)
```

Comment:
To genuinely check for all events, shouldn't we just not add this `CompareOperation` when `EventTaxonomyQuery.event` is "all events"?
https://github.com/PostHog/posthog/blob/82ddeda6f91c78ffd703785bad9cbed8cf12988a/posthog/hogql_queries/ai/event_taxonomy_query_runner.py#L113-L117